src/data/products-list.ts

```diff
@@ -0,0 +1,79 @@
+import { IProduct } from '@/types/IProduct'
+
+export const productsList: IProduct[] = [
```

Comment:
you have the wrong list of products the most simple example in types there is no title there is a name and so with the rest of the fields

Reply:
Done